{"name": "Проверка возвращении ошибки при отсутствии поля логина или пароля при авторизации", "status": "passed", "parameters": [{"name": "data", "value": "{'password': '122636'}"}], "start": 1717232909997, "stop": 1717232910620, "uuid": "0b806f43-9506-4f89-83d2-c71a2fb3e7a9", "historyId": "f8e5bf0d4aa17832d03266bb44ab1fbd", "testCaseId": "84c949e1be3456c467f0e0afc689978f", "fullName": "tests.test_login_courier.TestLoginCourier#test_post_login_courier_without_login_or_password_field_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "WIN-A5H5V66RP32"}, {"name": "thread", "value": "14664-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}